wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -487,6 +487,29 @@ public void createSnapshot(String workspaceId) throws NotFoundException, ServerE
         return machineManager.getSnapshots(sessionUser().getId(), workspaceId);
     }
 
+    /**
+     * Adds machine into workspace runtime.
+     *
+     * @param machine
+     *         machine to add to specified runtime
+     * @throws ConflictException
+     *         if given machine already exists in specified runtime
+     *         or if machine is dev-machine
+     * @throws NotFoundException
+     *         if machine's workspace hasn't runtime
```

Comment:
This sentence is quiet confusing, rephrase it. btw try to follow the class style in the case of documentation.